index.js

```diff
@@ -105,6 +105,16 @@ export class VFileMessage extends Error {
     // Feel free to add other non-standard fields to your messages.
 
     /* eslint-disable no-unused-expressions */
+    /**
+     * You may use this to specify the actual value that’s being reported.
+     * @type {string?}
+     */
+    this.actual
+    /**
+     * You may use this to suggest expected values that should be used instead of `actual`.
+     * @type {string[]?}
```

Comment:
```suggestion
     * @type {Array<string>?}
```

Tryin to be more strict about style here

Reply:
Can this be just a string too? I think so? Not sure? 🤔

Reply:
I want to go over all retext plugins soon to define `url`. I can make sure when I do.

Reply:
I checked all plugins listed [plugins.md](https://github.com/retextjs/retext/blob/main/doc/plugins.md). They all set `expected` as a string array.